package.json

```diff
@@ -31,15 +33,21 @@
     "jest": "^24.5.0",
     "jest-cli": "^24.5.0",
     "jest-styled-components": "^7.0.0",
+    "knex": "0.20.0",
     "lerna": "^3.13.1",
     "lint-staged": "^9.2.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-is": "16.8.0",
     "react-test-renderer": "^16.9.0",
+    "redux-saga": "1.0.0",
```

Comment:
Can you downgrade this version to `"redux-saga": "^0.16.0",` so it matches the strapi-admin and the strapi-plugin-users-permissions ones?